mmv1/third_party/terraform/services/apigee/resource_apigee_control_plane_access_test.go

```diff
@@ -0,0 +1,155 @@
+				ImportStateVerifyIgnore: []string{"name"},
+			},
+		},
+	})
+}
+
+func testAccApigeeControlPlaneAccess_full(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_project" "project" {
+  project_id      = "tf-test-my-project%{random_suffix}"
+  name            = "tf-test-my-project%{random_suffix}"
+  org_id          = "%{org_id}"
+  billing_account = "%{billing_account}"
+  deletion_policy = "DELETE"
+}
+
+resource "google_project_service" "apigee" {
+  project = google_project.project.project_id
+  service = "apigee.googleapis.com"
+}
+
+resource "google_apigee_organization" "apigee_org" {
+  analytics_region   = "us-central1"
+  project_id         = google_project.project.project_id
+
+  runtime_type       = "HYBRID"
+  depends_on         = [google_project_service.apigee]
+}
+
+resource "google_service_account" "service_account" {
+  account_id   = "sa-%{random_suffix}"
+  display_name = "Service Account"
+}
+
+resource "google_apigee_control_plane_access" "apigee_control_plane_access" {
+  name       = google_apigee_organization.apigee_org.name
+  synchronizer_identities = [
+    "serviceAccount:${google_service_account.service_account.email}",
+  ]
+  analytics_publisher_identities = [
+    "serviceAccount:${google_service_account.service_account.email}",
+  ]
+  depends_on = [google_service_account.service_account]
+}
+`, context)
+}
+
+func testAccApigeeControlPlaneAccess_update(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_project" "project" {
+  project_id      = "tf-test-my-project%{random_suffix}"
+  name            = "tf-test-my-project%{random_suffix}"
+  org_id          = "%{org_id}"
+  billing_account = "%{billing_account}"
+  deletion_policy = "DELETE"
+}
+
+resource "google_project_service" "apigee" {
+  project = google_project.project.project_id
+  service = "apigee.googleapis.com"
+}
+
+resource "google_apigee_organization" "apigee_org" {
+  analytics_region   = "us-central1"
+  project_id         = google_project.project.project_id
+
+  runtime_type       = "HYBRID"
+  depends_on         = [google_project_service.apigee]
+}
+
+resource "google_service_account" "ax_service_account1" {
+  account_id   = "sa-ax1%{random_suffix}"
+  display_name = "Service Account"
+}
+
+resource "google_service_account" "ax_service_account2" {
+  account_id   = "sa-ax2%{random_suffix}"
+  display_name = "Service Account"
+}
+
+resource "google_service_account" "sync_server_account1" {
+  account_id   = "sa-sync1%{random_suffix}"
+  display_name = "Service Account"
+}
+
+resource "google_service_account" "sync_server_account2" {
+  account_id   = "sa-sync2%{random_suffix}"
+  display_name = "Service Account"
+}
+
+resource "google_apigee_control_plane_access" "apigee_control_plane_access" {
+  name       = google_apigee_organization.apigee_org.name
+  synchronizer_identities = [
+    "serviceAccount:${google_service_account.ax_service_account1.email}",
```

Comment:
I would suggest testing the empty array case for both fields, maybe instead of the multiple accounts case, because zero values can do weird things in TF.